src/app/clusters/smoke-co-alarm-server/smoke-co-alarm-server.cpp

```diff
@@ -186,19 +185,19 @@ bool SmokeCoAlarmServer::SetDeviceMuted(EndpointId endpointId, MuteStateEnum new
     {
         AlarmStateEnum alarmState;
 
-        VerifyOrReturnValue(GetAttribute(endpointId, SmokeState::Id, SmokeState::Get, alarmState), false);
+        GetAttribute(endpointId, SmokeState::Id, SmokeState::Get, alarmState);
         VerifyOrReturnValue(alarmState != AlarmStateEnum::kCritical, false);
```

Comment:
Again, examining outparams if the getter failed makes no sense.  The `VerifyOrReturnValue` should be conditioned on the getter succeeding.

Even better would be if we only allowed failing due to "no such attribute" not general failures.  Especially here, where the failure mode is "we incorrectly allow things to be muted", which seems very bad.